README.md

```diff
@@ -23,6 +23,7 @@ https://yunohost.org/apps
 
  - **official.json** contains the repository information of validated apps.
  - **community.json** contains all references to known YunoHost packages. If you want to add your app to the list, please [send a Pull Request](#contributing)
```

Comment:
After that, will community.json and official.json not be deleted?

Reply:
Not in a first time.
It was said during the meeting, but nothing decided, that we would wait at least the version 4 of YunoHost + 6 months.

Reply:
Ok